docs/installation/install-with-configuration-file.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Deploying Karmada Using Configuration File with CLI
+---
+
+This documentation serves as a comprehensive guide for deploying Karmada using a configuration file through the `karmadactl` CLI tool.
+
+## Prerequisites
+
+- A Kubernetes cluster ready for Karmada installation.
+
+- A valid kube-config file to access your Kubernetes cluster.
+
+- The karmadactl command-line tool or kubectl-karmada plugin installed.
+
+  :::note
+
+  For installing `karmadactl` command line tool refer to the [Installation of CLI Tools](/docs/next/installation/install-cli-tools#one-click-installation) guide.
+
+  :::
```

Comment:
The `note` didn't render correctly. See the preview:  https://deploy-preview-690--karmada.netlify.app/docs/next/installation/install-with-configuration-file#deploying-karmada-with-configuration-file

Reply:
> The `note` didn't render correctly. See the preview: https://deploy-preview-690--karmada.netlify.app/docs/next/installation/install-with-configuration-file#deploying-karmada-with-configuration-file

I'm a bit unsure about what you meant by the note not rendering properly. I hope you can provide more details. On the page, I observed that this note should be viewable and navigable as expected.
![image](https://github.com/user-attachments/assets/bc9a3534-ca43-4102-a66b-9be5ee0eda58)

Reply:
It looks good now on my side. I don't know why.